apps/arm/include/CANConfig.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include "CANInterface.h"
+#include "CANMsg.h"
+#include "hw_bridge.h"
+
+static void sendACK(HWBRIDGE::ARM_ACK_VALUES ackValue);
+static mbed_error_status_t armSetControlMode(CANMsg& msg);
+static mbed_error_status_t armSetJointPIDParams(CANMsg& msg);
+static mbed_error_status_t commonSwitchCANBus(CANMsg& msg);
+
+namespace CANConfig {
+
+using namespace HWBRIDGE;
+
+static CANMsgMap rxStreamedMsgMap = {
+    {CANID::ARM_SET_JOINT_POSITION,
+     {
+         {CANSIGNAL::ARM_SET_TURNTABLE_POSITION, 0},
```

Comment:
I still haven't fleshed out the SNA handling for scaled signals, so leaving these as 0 for now. Some SNA checks are also missing right now